src/plays/pokemon-stats/PokemonStats.js

```diff
@@ -0,0 +1,70 @@
+import PlayHeader from "common/playlists/PlayHeader";
+import { useState } from "react";
+import "./styles.css";
+import axios from "axios";
+import Pokemoncard from "./components/Pokemoncard";
+import Search from "./components/Search";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+// WARNING: Do not change the entry componenet name
+function PokemonStats(props) {
+  // Your Code Start below.
+  const [pokemon, setpokemon] = useState();
+  const [name, setname] = useState("");
+  const fetchPokemonDetails = (e) => {
+    e.preventDefault();
+    axios
+      .get(`https://pokeapi.co/api/v2/pokemon/${name.toLowerCase()}`)
+      .then((response) => {
+        setpokemon(response.data);
+      })
+      .catch((err) => {
+        toast.error("Sorry, Unable to fetch", {
+          position: "bottom-left",
+          autoClose: 2000,
+          hideProgressBar: false,
+          closeOnClick: true,
+          pauseOnHover: true,
+          draggable: true,
+          progress: undefined,
+        });
+      });
+  };
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={props} />
+        <div className="Skeleton-pokemon-app">
+          {/* Your Code Starts Here */}
+          <ToastContainer
```

Comment:
Some random text search didn't show me the toast however the console is recording the problem
![image](https://user-images.githubusercontent.com/7145967/188255875-2e96acd4-87e9-4d67-898a-d002dedc419c.png)